src/lib/transition_frontier_controller_tests/stubs.ml

```diff
@@ -361,14 +361,18 @@ struct
 
   module Ledger_transfer = Coda_base.Ledger_transfer.Make (Ledger) (Ledger.Db)
 
-  let genesis_frontier ~logger =
-    let ledger_db = Coda_base.Ledger.Db.create () in
+  let with_genesis_frontier ~logger =
+    let%bind ledger_dir =
```

Comment:
can you make sure this gets deleted at the end?

```
let with_genesis_frontier ~logger ~f =
  let%bind ledger_dir = (* ... *) in
  let%bind () = f ledger_dir in
  delete_directory ledger_dir
```

Something like that ^^

Otherwise looks good